File: job_array/Array.java
import java.lang.Thread;
public class HelloWorld {
	public static void main(String [] args){
		for (int i=0; i<5; i++){
			System.out.println("Hello World! I am at step " + i + " of the loop in index #" + System.getenv("SLURM_ARRAY_TASK_ID") + "!");
		}
	}
}

File: job_array/array.py
import time
import os

ARRAY_TASK_ID = os.getenv('SLURM_ARRAY_TASK_ID')

for i in range(1,5):
	time.sleep(60)
	if (i == 1):
		print(f'{i} minute has elapsed in task {ARRAY_TASK_ID}.')
	else:
		print(f'{i} minutes have elapsed in task {ARRAY_TASK_ID}.')
print("Hello World!")
